src/UI/Component/Chart/Speedo/Factory.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/* Copyright (c) 2017 Ralph Dittrich <dittrich@qualitus.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Chart\Speedo;
+
+/**
+ * Interface Factory
+ * @package ILIAS\UI\Component\Chart\Speedo
+ */
+interface Factory {
+
+    /**
+     * ---
+     * description:
+     *   purpose: The standard speedo is used if the speedo is to be rendered in it's original size.

```

Comment:
Add something like, "even on mobile devices". Further this is a little dim. Go further into the details of what this diagnostic is all about. E.g.:
It extends the visualisation by allowing to set a diagnostic value representing the score of some ... (for you to be completed, what is this used for?).